teams.md/static/missing-pages.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
This file assists with navigation. If user tries to switch languages via the dropdown, but the new language doesn't have the same page, the following will happen:
- Redirect canceled
- language context restored to the original language
- Banner will show up saying the current page doesn't exist for {new language} context. Do you want to redirect to {new language}'s getting started docs?
    - If user presses button, they will be taken to `teams-ai/{newLanguage}/getting-started` page.
    - If they do not interact or press close button, banner will disappear.